src/helpers/helpers.canvas.js

```diff
@@ -65,17 +66,16 @@ var exports = module.exports = {
 		ctx.save();
 		ctx.translate(x, y);
 		ctx.rotate(rotation * Math.PI / 180);
+		ctx.beginPath();
```

Comment:
We don't need `closePath()` before `stroke()`. [[link](https://stackoverflow.com/questions/22432036/do-i-have-to-have-the-content-beginpath-and-content-closepath)]

In the image for #5629, you see some lines are darker than others. This is because the last line in a path is rendered twice as a result of closing the subpath. This can be avoided by not calling `closePath()`.
![image](https://user-images.githubusercontent.com/723188/42519301-5347c3d8-8496-11e8-9908-4cfa85be9657.png)